assets/src/dashboard/templates/index.js

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
```

Comment:
remove?

Reply:
Good catch, I fixed it, not sure how that got in there.